React/CxxBridge/RCTCxxBridge.mm

```diff
@@ -112,9 +112,9 @@ static void notifyAboutModuleSetup(RCTPerformanceLogger *performanceLogger, cons
   }
 }
 
-static void registerPerformanceLoggerHooks(RCTPerformanceLogger *performanceLogger) {
+static ReactMarker::LogTaggedMarker registerPerformanceLoggerHooks(RCTPerformanceLogger *performanceLogger) {
```

Comment:
Comment from @mhorowitz 

This is also a strange name, since it's not "registering" anymore.

Reply:
I've made it `getPerformanceLoggerHooks` now. Let me know if that's okay.